packages/front-end/components/Settings/NewExperimentChecklistItem.tsx

```diff
@@ -0,0 +1,145 @@
+    {
+      value: "Upload a screenshot for each variation of the experiment",
+      label: "Upload a screenshot for each variation of the experiment",
+      propertyKey: "screenshots",
+    },
+    {
+      value: "Add a description for this experiment",
+      label: "Add a description for this experiment",
+      propertyKey: "description",
+    },
+    {
+      value: "Add this experiment to a project",
+      label: "Add this experiment to a project",
+      propertyKey: "project",
+    },
+    {
+      value: "Add atleast 1 tag to this experiment",
+      label: "Add atleast 1 tag to this experiment",
+      propertyKey: "tag",
+    },
+  ];
+
+  function addNewTask(newTaskInput: ChecklistTask) {
+    setExperimentLaunchChecklist([...experimentLaunchChecklist, newTaskInput]);
+    setNewTaskInput(undefined);
+  }
+
+  return (
+    <div className="pt-5 pb-2">
+      <h4>New Task</h4>
+      <div className="p-3 border rounded">
+        <label>Task</label>
+        <CreatableSelect
+          className="pb-3"
+          options={autoChecklistOptions.filter((option) => {
+            return !experimentLaunchChecklist.some(
+              (index) => index.task === option.value
+            );
+          })}
+          placeholder="Choose from pre-defined tasks or create your own custom task"
+          onChange={(option: AutoChecklistOption) => {
+            setNewTaskInput({
+              task: option.value,
+              completionType: "auto",
+              propertyKey: option.propertyKey,
+            });
+          }}
+          onCreateOption={(inputValue) => {
+            setNewTaskInput({
+              task: inputValue,
+              completionType: "manual",
+            });
+          }}
+          noOptionsMessage={() =>
+            "No more pre-defined tasks available. Start typing to create a new task"
+          }
+          value={
+            newTaskInput.task
+              ? {
+                  label: newTaskInput.task,
+                  value: newTaskInput.task,
+                  propertyKey: newTaskInput.propertyKey,
+                }
+              : null
+          }
+          {...ReactSelectProps}
+        />
+        {newTaskInput.task && newTaskInput.completionType === "manual" ? (
+          <Field
+            label="Add URL (Optional)"
+            autoFocus
+            type="url"
+            helpText="Add a URL to this task to help your team complete this task."
+            {...newTaskInput}
+            {...newTaskInput}
+            onKeyDown={(e) => {
+              if (e.key === "Enter") {
+                // Prevent this action from closing the modal
+                e.preventDefault();
+                addNewTask(newTaskInput);
+              }
+            }}
+            onChange={(e) => {
+              const url = e.target.value.toLowerCase();
+
+              const containsHttp =
+                url.substring(0, 8) == "https://" ||
+                url.substring(0, 7) == "http://";
```

Comment:
just FYI other ways to do this more concisely
```suggestion
              const containsHttp = url.startsWith('http://') || url.startsWith('https://') ;
```
or
```suggestion
              const containsHttp = (/^http(s)?:\/\//).test(url)
```

Reply:
TY! I think your first suggestion is a bit more readable. Going with that one.